### DISCUSSION TITLE
Video isn't loading

### DISCUSSION THREAD

**Discussion by nawed2611** [Chatmosphere/chatmosphere-app#187] [Q&A]
Any idea on how to fix this?

<img width="326" alt="image" src="https://user-images.githubusercontent.com/83456083/167243328-8de6bad8-d0f2-4b99-b458-19bd2a7d2413.png">

**Comment by dkgrieshammer** [comment]
Hey @nawed2611 can you elaborate a little more on your setup? Which Browser are you on and what system? Did you allow camera access yet? thx & best, David

**Comment by nawed2611** [comment]
When I join a meeting call, sometimes the video doesn't load properly in a few cases. Yes, I did allow camera access

I'm using Google Chrome as a browser and a windows system.
